### PR TITLE
Feature/creatures redirection

### PR DESCRIPTION
**Pull request by Alejoboga20**
### Description: 

When a goes into https://creaturechronicles.io/ we need to send him or her to /creatures page and disable categories in top bar to avoid navigation.

- Creation of custom hook to handle redirection.
- Creation of new route based in user type.
- Refactoring top bar component to use custom hook instead of props to disable categories.

Reviewers:
@natokra @gastonri @kevin-rodriguez @ImanolH96 @acastelli 

Screenshots:
image

### CODE REVIEW

File: src/app/router/index.tsx [hunk 1]
```diff
@@ -6,9 +7,62 @@ import CreaturesPage from 'presentation/pages/Categories/Creatures/CreaturesPage
...
+      path: routesPaths.my_collection,
+      component: <MyCollection />,
+      private: true,
+    },
+  ],
+};
+/* 
+
+[
   {
```
Comment (natokra) [inline src/app/router/index.tsx:66]: you have commented code here

**Review by natokra — APPROVED** [review]
Good job! just a little thing!

File: src/app/router/index.tsx (hunk 1 above)
Comment (Alejoboga20) [reply src/app/router/index.tsx:66]: thanks. I will remove it

**Review by natokra — APPROVED** [review]

**Review by gastonri — APPROVED** [review]
Nice way of handling the redirection! 🚀

**Pull request merged**